### DISCUSSION TITLE
tidak bisa memenampilkan gambar pada saat preview

### DISCUSSION THREAD

**Discussion by DAffaa-37** [deaafrizal/forum#94] [Troubleshooting Room]
![code2](https://github.com/deaaprizal/forum/assets/175019963/dda495ed-8f2d-4157-8ef1-9dd89fbea7b2)
![vscode folder](https://github.com/deaaprizal/forum/assets/175019963/5848a1fd-31af-4b62-a160-50ef8d2fdeb2)

**Comment by DAffaa-37** [comment]
![preview](https://github.com/deaaprizal/forum/assets/175019963/383af324-9939-4696-b351-0adedba14abe)

**Reply by DAffaa-37** [reply]
Pada saat preview keluarnya seperti itu

**Comment by TEUNGKU-ZULKIFLI** [comment]
# answer with my opinion

https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEhw9TGppVjAQ3t0dHualV-wNiHzisKdwF8WJitzjdv8RHxA1FXiZam8TzS0-CN3hLOBzoyM_HPUiCGeuMD-MtoXyaBVxfhPmLVlbYRi7gQ5-2wYJy9i0IN2lR4kHnS1F8eo1c-msb85dP2CX_DrmsUzpmvUT2vjOfcZY12OExET_gvA0oL9PHi4MowybaE/s16000/fixed%20name%20path.gif
---
